libraries/standard/mqtt/utest/mqtt_utest.c

```diff
@@ -1260,3 +1282,308 @@ void test_MQTT_ProcessLoop_Timer_Overflow( void )
     mqttStatus = MQTT_ProcessLoop( &context, MQTT_TIMER_OVERFLOW_TIMEOUT_MS );
     TEST_ASSERT_EQUAL( MQTTSuccess, mqttStatus );
 }
+
+/* ========================================================================== */
+
+/**
+ * @brief This test case verifies that MQTT_Subscribe returns MQTTBadParameter
+ * with an invalid parameter. This test case also gives us coverage over
+ * the private method, validateSubscribeUnsubscribeParams(...).
+ */
+void test_MQTT_Subscribe_invalid_params( void )
+{
+    MQTTStatus_t mqttStatus;
+    MQTTContext_t context;
+    MQTTSubscribeInfo_t subscribeInfo;
+
+    /* Call subscribe with a NULL context. */
+    mqttStatus = MQTT_Subscribe( NULL, &subscribeInfo, 1,
+                                 MQTT_FIRST_VALID_PACKET_ID );
```

Comment:
These parameters should either be all on one line, or one per line

Reply:
Sure